rest/src/main/groovy/whelk/rest/api/ESQuery.groovy

```diff
@@ -125,55 +177,65 @@ class ESQuery {
         return query
     }
 
+    List<String> getBoostFields(String[] types, String boostMode) {
+        if (boostMode?.indexOf('^') > -1) {
+            return boostMode.tokenize(',')
+        }
+
+        String typeKey = types != null ? types.toUnique().sort().join(',') : ''
+        typeKey += boostMode
+        List<String> boostFields = boostFieldsByType[typeKey]
+        if (boostFields == null) {
+            boostFields = boostFieldsByType[typeKey] =
```

Comment:
I feel this would be more readable with an `if` rather than using the ternary operator.

Reply:
Fair point; changing!